x/treasury/types/params.go

```diff
@@ -46,6 +47,7 @@ var (
 	DefaultTaxRate                 = sdk.NewDecWithPrec(1, 3)   // 0.1%
 	DefaultRewardWeight            = sdk.NewDecWithPrec(5, 2)   // 5%
 	DefaultBurnTaxSplit            = sdk.NewDecWithPrec(5, 1)   // 50%
+	DefaultMinInitialDepositRatio  = sdk.ZeroDec()
```

Comment:
Second question: shouldn't this one changeable through gov? else upon upgrade, 0*MinInitialDeposit = 0 won't change anything.

Also, I think DefaultMinInitialDepositRatio should equal a value > 0 like 0.5

Reply:
@fragwuerdig

Reply:
I think the initial value cannot be > 0 as it was not decided by governance to have a minimum **initial** amount to spend.

Reply:
@nghuyenthevinh2000 this should be changeable through gov, isn't it? I thought the way I implemented it would make it changeable through gov? Please, let me know.

Reply:
It looks to me that all parameters in the paramSpace of a keeper are changeable through the parameter module. I didn't see any extra handlers for the other parameters in the treasury module. So I just assumed it work for that one too. But I probably should test it in a local testnet.

Reply:
maybe I should do a general testing framework for gov